src/main/java/com/exasol/report/ValidationReport.java

```diff
@@ -0,0 +1,32 @@
+package com.exasol.report;
+
+/**
+ * Represents a validation report.
+ */
+// [impl->dsn~rr-creates-validation-report~1]
+public class ValidationReport extends AbstractReport {
+    /**
+     * Register a new successful validation.
+     * 
+     * @param validatedContent description of the validated content
+     */
+    public synchronized void addSuccessfulValidation(final String validatedContent) {
```

Comment:
Why is it synchronized? Are you validating in parallel threads?

Reply:
I just want to indicate that this place is not safe for the future